test/connect.test.ts

```diff
@@ -0,0 +1,136 @@
+import Ipfs from 'ipfs-core'
+import { IpfsDaemon } from '@ceramicnetwork/ipfs-daemon'
+import { IpfsTopology } from '@ceramicnetwork/ipfs-topology'
+import { DiagnosticsLogger } from '@ceramicnetwork/common'
+
+import peerlistDevUnstable from '../dev-unstable.json'
+import peerlistTestnetClay from '../testnet-clay.json'
+import peerlistMainnet from '../mainnet.json'
+
+const ceramicNetwork: string = process.env.JEST_CERAMIC_NETWORK || ''
+
+const peerlistByNetwork: { [key: string]: string[] } = {
+    'dev-unstable': peerlistDevUnstable,
+    'testnet-clay': peerlistTestnetClay,
+    'mainnet': peerlistMainnet
+}
+
+const tempFilePath = () => `/tmp/jest/peerlist/connect.test.ts/${Date.now()}`
+
+describe('IpfsDaemon', () => {
+    let ipfsDaemon;
+    afterEach(async () => {
+        await ipfsDaemon.stop()
+    })
+    test('fails to connect to invalid bootstrap list', async () => {
+        const consoleSpy = jest.spyOn(console, 'log')
+
+        const peerlist = ['not.a.multiaddress']
+        ipfsDaemon = await createIpfsDaemon(peerlist)
```

Comment:
This could happen in `beforeEach` I guess.

Reply:
The peerlist that gets passed in is different for each so that's why i didn't pull in there